afs/afs-storage-api/src/main/java/com/powsybl/afs/storage/NodeGenericMetadata.java

```diff
@@ -0,0 +1,121 @@
+/**
+ * Copyright (c) 2017, RTE (http://www.rte-france.com)
+ * This Source Code Form is subject to the terms of the Mozilla Public
+ * License, v. 2.0. If a copy of the MPL was not distributed with this
+ * file, You can obtain one at http://mozilla.org/MPL/2.0/.
+ */
+package com.powsybl.afs.storage;
+
+import java.util.HashMap;
+import java.util.Map;
+import java.util.Objects;
+
+/**
+ * @author Geoffroy Jamgotchian <geoffroy.jamgotchian at rte-france.com>
+ */
+public class NodeGenericMetadata {
```

Comment:
For stringMetadata, null values are allowed but not for the other metadatas. Is it OK ?

Reply:
No, I added to check to not allow null value